tasks/tvheadend.yml

```diff
@@ -0,0 +1,51 @@
+# Some indirection to be able to extend the devices list conditionally
+- name: Define basic docker devices
+  set_fact:
+    tvheadend_docker_devices: []
+
+- name: Add hw acceleration
+  set_fact:
+    tvheadend_docker_devices: "{{ tvheadend_docker_devices + [ '/dev/dri:/dev/dri' ] }}"
+  when: tvheadend_hw_accel
+
+- name: Add dvb hw
```

Comment:
This is a little 'princess' comment but it seems a bit cryptic with two abbreviations in the name. Maybe something along the lines of 'Add hardware TV Tuner' or 'Mount hardware TV Tuner' would be a little more accessible.